src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -101,6 +120,20 @@ const SidebarLinks = (props) => {
                     isChatSwitcherActive={props.isChatSwitcherActive}
                 />
             </View>
+            <View style={[
+                styles.flexRow,
+                styles.sidebarHeaderTop,
+                styles.justifyContentBetween,
+                styles.alignItemsCenter,
+            ]}
+            >
+                <Header textSize="large" title="Chats" />
+                <AvatarWithIndicator
+                    indicatorPosition="bottom-right"
```

Comment:
We can remove this now.

Reply:
This is the main code. Why do you want to remove it?

Reply:
Understood you mean the prop